examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/MCCastingApp.h

```diff
@@ -42,6 +42,16 @@
  */
 - (NSError * _Nullable)initializeWithDataSource:(id<MCDataSource> _Nonnull)dataSource;
 
+/**
+ * @brief Updates the MCCommissionableDataProvider stored in this CastingApp with the new CommissionableData
+ * to be used for the next commissioning session. Calling this function is mandatory for the
+ * Casting Player/Commissioner-Generated passcode commissioning flow, since the commissioning session's PAKE
+ * verifier needs to be updated with the user entered passcode.
+ *
+ * @param newCommissionableData the new MCCommissionableData to be used for the next commissioning session.
+ */
+- (NSError * _Nullable)updateCommissionableDataProvider:(MCCommissionableData * _Nonnull)newCommissionableData;
```

Comment:
Ok, we can do that for the sake of consistency. updateCommissionableDataProvider() will be called from MCCastingPlayer.continueConnecting(). We'll store the MCAppParametersDataSource in an singleton instance of the MCInitializationExample so that can be accessed and updated by the client form MCConnectionExampleViewModel.swift (prior to calling MCCastingPlayer.continueConnecting()).